spicepod.yaml

```
@@ -0,0 +1,104 @@
+version: v1beta1
+kind: Spicepod
+name: spice-oss-docs
+
+datasets:
+  - from: github:github.com/spiceai/docs/files/trunk
+    name: spiceai.docs
```

If this is the only thing, why not just put this in the default schema?

```suggestion
    name: docs
```